examples/README.md

```diff
@@ -1 +0,0 @@
-The examples in this folder are made based on the OCaml's toplevel and Owl's Zoo system. Please refer to [Tutorial 9: Scripting and Zoo System](https://github.com/ryanrhymes/owl/wiki/Tutorial:-Scripting-and-Zoo-System).
```

Comment:
Could you keep the file, adding a couple of lines on how to run the examples? There is no dune file in the folder, so a priori is not obvious for a new user what to do now.

Reply:
Thanks! There is actually a dune file added to this folder, and I've updated the readme file accordingly.